### PR TITLE
PHOENIX-6594 Clean up vararg warnings flagged as errors by Eclipse

### PR DESCRIPTION
**Pull request by stoty**

### CODE REVIEW

File: phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexToolUtil.java [hunk 1]
```diff
@@ -74,7 +74,7 @@ public static void updateIndexState(Connection connection, final String masterTa
         Preconditions.checkNotNull(connection);
         final String alterQuery = String.format(ALTER_INDEX_QUERY_TEMPLATE,indexTable,masterTable,state.name());
         connection.createStatement().execute(alterQuery);
-        LOGGER.info(" Updated the status of the index {} on {} to {} ", indexTable , masterTable, state.name());
+        LOGGER.info(" Updated the status of the index {} on {} to {} ", indexTable ,masterTable, state.name());
```
Comment (richardantal) [inline phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexToolUtil.java:77]: nit: whitespace should be after the comma

Comment (stoty) [reply phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexToolUtil.java:77]: Thanks, I'll fix that on commit.

Comment (stoty) [reply phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexToolUtil.java]: Actually, I've just pushed the fix here.

**Review by richardantal — APPROVED** [review]
LGTM

**Pull request closed without merging**